src/api.js

```diff
@@ -782,6 +782,13 @@ API.prototype.declare = function(options, handler) {
     options.output = this._options.schemaPrefix + options.output;
   }
   this._entries.push(options);
+  var valueArr = options.map((item) => {return item.name;});
```

Comment:
`options` here is an object with properties -- there's no need to use `map` to find its name attribute.  Just write `options.name`.